helm/upp-aggregate-healthcheck/app-configs/upp-aggregate-healthcheck_eks_delivery.yaml

```diff
@@ -151,7 +225,15 @@ categories:
     immutable: false
     data:
       category.name: annotations-publish
-      category.services: public-annotations-api,system-healthcheck
+      category.services: >-
+        pac-annotations-mapper,
+        upp-next-video-annotations-mapper,
+        annotations-rw-neo4j,
+        annotations-writer-ontotext,
```

Comment:
`annotations-writer-ontotext` is part of the automatic (V2) annotations flow. It is triggered by a content publication. So if it needs to be in a category it probably should be in the `content-publish` or a new one. The rest are part of the manual annotation publications. 

The previous behaviour for `annotations-publish` and `annotations-read`, we've changed the meaning of both categories. In the old setup `publish` was looking for `public-annotations-api` and `read` was looking for writers and mappers. 
I agree with the change, as it is logical, given the category names, for `annotations-read` to look for `public-annotations-api`. But by that logic `public-annotations-api` shouldn't be part of `annotations-publish`.

Reply:
Hmm, I added `annotations-writer-ontotext` based on the Kafka [diagram](https://lucid.app/lucidchart/72544b1a-9ed2-4b0b-90c5-fb749c6b7fda/edit?page=D0i.FFbelbdG&invitationId=inv_8845e9ff-a37f-4c7b-9cd4-d8568327064d#)

`public-annotations-api` wasn't removed because it's a dependency of the `annotations-writer-ontotext`. Will remove both.

Reply:
My mistake `annotations-writer-ontotext` was correct to be in `annotations-publish`. I was thinking of `concept-suggestion-ontotext`